src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/StreamExtensions.cs

```diff
@@ -0,0 +1,33 @@
+#nullable enable
+using System;
+using System.IO;
+using System.Security.Cryptography;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Extensions
+{
+    public static class StreamExtensions
+    {
+        public static void WriteText(this Stream stream, string text)
+        {
+            using var writer = new StreamWriter(stream, leaveOpen: true);
+            writer.Write(text);
+            writer.Flush();
+        }
+
+        public static string ReadToEnd(this Stream stream)
+        {
+            using var reader = new StreamReader(stream, leaveOpen: true);
+            return reader.ReadToEnd();
```

Comment:
Cool, seems like a nice addition to test stream content is correct within unit tests! 🙂